test/app.spec.js

```diff
@@ -0,0 +1,52 @@
+import chai, { expect } from 'chai'
```

Comment:
You see you are using different test file one is test.js which is from our develop and then app.spec.js 
you can continue using test.js

Reply:
Now working thanks